package.json

```diff
@@ -16,59 +16,63 @@
     "test": "ember test"
   },
   "devDependencies": {
-    "@ember/jquery": "0.6.0",
+    "@glimmer/component": "^1.0.0",
     "ajv": "^6.10.0",
+    "babel-eslint": "^10.0.3",
     "clean-css": ">=4.1.11",
     "coveralls": "^3.0.2",
-    "ember-auto-import": "1.2.21",
+    "ember-ajax": "^5.0.0",
+    "ember-auto-import": "^1.5.3",
     "ember-cli-code-coverage": "^0.4.2",
     "ember-cli-mirage": "^1.1.1",
+    "ember-cli-qunit": "^4.3.2",
+    "ember-concurrency": "^1.1.6",
+    "ember-fetch": "^7.0.1",
     "ember-load": "0.0.12",
     "ember-lodash": "^4.19.4",
-    "ember-qunit": "^4.5.1",
+    "ember-qunit": "^4.6.0",
     "ember-test-selectors": "^2.1.0",
     "ember-tether": "^1.0.0",
+    "eslint-plugin-ember": "^7.9.0",
     "merge": ">=1.2.1",
-    "qunit-dom": "^0.6.3",
+    "qunit-dom": "^0.9.2",
     "toastr": "^2.1.4"
   },
   "engines": {
-    "node": "^4.5 || 6.* || >= 7.*"
+    "node": "6.* || 8.* || >= 10.*"
   },
   "dependencies": {
```

Comment:
This is fine for this round. Eventually we should move as much as we can to `devDependencies` instead of `dependencies`